Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -315,6 +319,17 @@ public void serverStatus(@Nullable final AnalysisStatus analysisStatus, @Nullabl
     }
   };
 
+  public static boolean doEnableAnalyzedFilesSubscription() {
+    PropertiesComponent properties = PropertiesComponent.getInstance();
```

Comment:
Yes, I changed it yesterday from being a project level setting to a global setting, I squashed the commits together so you can't see the history.

I figured that if someone is enabling/disabling it they probably want it configured everywhere, also by not having it be a project setting, the UI will be easier and cleaner to implement.  I changed the source yesterday morning but the GitHub web UI was down so I couldn't change the text in the PR, updating it now.  Updating the method name now.  I think it does need to be public to be called by third party plugins.